.travis.yml

```diff
@@ -14,5 +17,6 @@ cache:
     - $HOME/.sbt/boot/
 script:
   - sbt ++$TRAVIS_SCALA_VERSION validate
+# - "./build/build.sh"
```

Comment:
Maybe we should just give up on built matrix and start using `build.sh`? It seems to be what Flavio is doing for Arrows (https://github.com/traneio/arrows/blob/master/.travis.yml).

Reply:
makes sense to me!

Reply:
@fwbrasil i see you went through a couple of different iterations on doing cross builds to finally hardcoding `2.12` in your build script. Any helpful gotchas that you discovered in the process?